src/leap/keymanager/__init__.py

```diff
@@ -219,13 +220,21 @@ def _fetch_keys_from_server(self, address):
             res = self._get(self._nickserver_uri, {'address': address})
             res.raise_for_status()
             server_keys = res.json()
+
             # insert keys in local database
             if self.OPENPGP_KEY in server_keys:
+                # nicknym server is authoritative for its own domain,
+                # for other domains the key might come from key servers.
+                validation_level = ValidationLevel.Weak_Chain
```

Comment:
It's an enum, it's lower case: https://github.com/leapcode/keymanager/blob/develop/src/leap/keymanager/validation.py#L30